modules/pel/peltool/peltool.py

```diff
@@ -372,14 +403,12 @@ def main():
                         help='Delete original file after parsing')
     parser.add_argument('-t', '--termination', dest='critSysTerm',
                         action='store_true', help='List only critical system terminating PELs')
-    if inBMC:
-        parser.add_argument('-l', '--list',
-                            help='List PELs',
-                            action='store_true')
-    else:
-        parser.add_argument('-l', '--list',
-                            help='List PELs',
-                            dest='list')
+    parser.add_argument('-i', '--id',
+                        dest='pelID', help='Display a PEL based on its ID')
+    parser.add_argument('-P', '--path',
```

Comment:
Should we only include this in environments that are not BMC-based?

I would push a separate commit for the -P option rather than merging it with -a. In this separate commit, we can also adjust the list option if needed. However, it's entirely up to you.